### DISCUSSION TITLE
How long until version 0.4 is released?

### DISCUSSION THREAD

**Discussion by bluehysteric** [ivangabriele/clamav-desktop#130] [Questions & Answers]
How long until version 0.4 is released?

**Comment by bluehysteric** [comment]
How long until version 0.4 is released?

**Comment by ivangabriele** [comment]
Duplicate of #87.

**Comment by Neustradamus** [comment]
Linked to:
- https://github.com/ivangabriele/clamav-desktop/issues/235
- https://github.com/ivangabriele/clamav-desktop/discussions/237
- https://github.com/ivangabriele/clamav-desktop/issues/216
- https://github.com/ivangabriele/clamav-desktop/discussions/239
- https://github.com/ivangabriele/clamav-desktop/issues/211
- https://github.com/ivangabriele/clamav-desktop/discussions/238
- https://github.com/ivangabriele/clamav-desktop/issues/187
- https://github.com/ivangabriele/clamav-desktop/discussions/190
- https://github.com/ivangabriele/clamav-desktop/issues/180
- https://github.com/ivangabriele/clamav-desktop/discussions/188
- https://github.com/ivangabriele/clamav-desktop/issues/127
- https://github.com/ivangabriele/clamav-desktop/discussions/131
- https://github.com/ivangabriele/clamav-desktop/issues/118
- https://github.com/ivangabriele/clamav-desktop/discussions/130
- https://github.com/ivangabriele/clamav-desktop/issues/85
- https://github.com/ivangabriele/clamav-desktop/discussions/87
